accname/name/comp_name_from_content.tentative.html

```diff
@@ -37,6 +37,23 @@ <h3 data-expectedlabel="image link" data-testname="heading with link referencing
   <img id="nested_image_label4" alt="image" aria-labelledby="nested_image_label4 crossref_link2" src="data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==">
 </h3>
 
+
+<!-- “びょういんのかんじゃサービス” means “Hospital patient services” -->
+<!-- text-transform:full-size-kana incorrectly changes: -->
+<!--   1. びょういん (byōin = hospital) into びよういん (biyōin = beauty parlor) -->
+<!--   2. かんじゃ (kanja = patient) into かんじや (kanjiya = [no meaning]) -->
+<!-- ...causing the whole heading to become nonsensical. -->
+<h1>heading which visually transforms <span lang="ja">びょういん</span> (hospital) to <span lang="ja">びよういん</span> (beauty parlor) using CSS <code>text-transform:full-size-kana</code></h1>
+<h3 lang="ja" data-expectedlabel="びょういんのかんじゃサービス" data-testname="heading name from content with text-transform:full-size-kana" class="ex" style="text-transform:full-size-kana;">びょういんのかんじゃサービス</h3>
+
+<!-- “竜の背に乗る” means “Ride the dragon’s back” -->
+<!-- text-transform:full-size-kana incorrectly changes 竜’s <rt> furigana: -->
```

Comment:
"incorrectly" may be confusing... As it's the correct behavior for rendered text, but should not be transformed in the test exposed to the Accessibility APIs… Note: I expect implementors may have questions about how to achieve this, so it's good this is in a tentative file.

Reply:
Here’s [another cut at that](https://github.com/web-platform-tests/wpt/pull/47535/files#diff-974c66e8b6fda730bf40c4e0b0ae3750f4dd71a89e5ee74f0b3dcb10f36900bcR42). I also went ahead and commented out [the second subtest](https://github.com/web-platform-tests/wpt/pull/47535/commits/4d5410bfe7eb96849a4261ca95d8560146374930).